node/lib/util/submodule_change.js

```diff
@@ -45,20 +45,31 @@ class SubmoduleChange {
      * null `oldSha` implies that the submodule was added, a null `newSha`
      * implies that it was removed, and if neither is null, the submodule was
      * changed.
+     * 
+     * UPDATE: the change used to contain only their sha, because default our
```

Comment:
Don't write comments that amend other comments.  Instead, just rewrite the other comments.